x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/common/card_selector_list.styles.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { css } from '@emotion/css';
+import { useEuiTheme, useEuiBackgroundColor } from '@elastic/eui';
+
+export const useCardSelectorListStyles = () => {
+  const { euiTheme } = useEuiTheme();
+  const itemBackgroundColor = useEuiBackgroundColor('primary');
```

Comment:
This is deprecated in the EUI refresh, new options: `Background colors` in https://github.com/elastic/kibana/issues/199715#issuecomment-2551952378